bsg_fpu/bsg_fpu_pkg.vh

```diff
@@ -5,14 +5,12 @@
  *
  */
 
-package bsg_fpu_pkg;
-
+`ifndef BSG_FPU
```

Comment:
Please rename this file to "bsg_fpu_defines.vh", and use the macro "BSG_FPU_DEFINES_VH" for header guard.